app/server/auth.tsx

```diff
@@ -24,6 +25,7 @@ export const auth = betterAuth({
     expiresIn: envServer.SESSION_EXPIRATION_IN_SECONDS,
     updateAge: envServer.SESSION_UPDATE_AGE_IN_SECONDS,
   },
+  trustedOrigins: [envServer.AUTH_TRUSTED_ORIGIN],
```

Comment:
A comment to explain the purpose of this line would be great so people not using Start UI [native] will know that they can remove it.